cardano_node_tests/tests/tests_conway/test_no_confidence.py

```diff
@@ -333,7 +333,6 @@ def test_no_confidence_action(
     @allure.link(helpers.get_vcs_link())
     @pytest.mark.skipif(not configuration.HAS_CC, reason="Runs only on setup with CC")
     @pytest.mark.long
-    @pytest.mark.disabled(reason="Needs to be fixed in 9.0.0+")
```

Comment:
The `test_committee_min_size` is broken in 9.0.0+.

Reply:
I've just fixed it in https://github.com/IntersectMBO/cardano-node-tests/pull/2511